conftest.py

```diff
@@ -15,5 +15,5 @@
 #
 import pytest
 
-pytest_plugins = ["jupyter_server.pytest_plugin"]
+#pytest_plugins = ["jupyter_server.pytest_plugin"]
```

Comment:
Is this causing issues?  I _think_ it's necessary for tests that use fixtures from jupyter_server.  The metadata tests use a number of these.  That said, I think I've found this to be unnecessary, but that never made sense to me - so I'm curious if you found some adverse to defining this.  (The pytest fixture stuff is still a bit of magic to me.)

Reply:
My mistake, I left this in there. I've had to comment this out when testing locally due to an error being thrown when using pytest. Let me re-enable it. 
I'm kind of in the same boat you are, this file has been unnecessary for me but needed for others.